cities/rheinstetten.json

```diff
@@ -0,0 +1,34 @@
+{
+  "crs": {
+    "properties": {
+      "name": "urn:ogc:def:crs:OGC:1.3:CRS84"
+    },
+    "type": "name"
+  },
+  "features": [
+    {
+      "geometry": {
+        "coordinates": [
+            48.974798,
+            8.312010
```

Comment:
The coordinates look a bit off:

![image](https://github.com/user-attachments/assets/33f8953c-01f8-4656-8f89-d5366183ed4b)

Reply:
:heavy_check_mark: Thanks for the fix.

Reply:
Yes, lat lon are the wrong way. 
Since in cities.json these are the other way (have to be switched), I guess that could be worth a bug, since it is unexpected? However, for Germany lat lon are easy to get right (by looking at an example) due to their large difference (ranges: lon 8-12 lat 45-54).

```
cities/cities.json:    "karlsruhe": {
cities/cities.json:        "id": "karlsruhe",
cities/cities.json:        "label": "Karlsruhe",
cities/cities.json-        "coordinates": [
cities/cities.json-            49.0092097,
cities/cities.json-            8.4039514

cities/karlsruhe.json-            "geometry": {
cities/karlsruhe.json-                "coordinates": [
cities/karlsruhe.json-                    8.40384,
cities/karlsruhe.json-                    49.00916
```

Reply:
corrected by the force-push

Reply:
Ohh :astonished: What a nice bug. Good catch. Yes, please file a bug here. I never noticed. I guess some processing code has to be fixed, too. Without looking into it: I suppose the order is based on what Leaflet likes.